src/Vts.Scripting/IDemoScript.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Vts.Scripting;
+
+/// <summary>
+/// Interface to constrain all demo scripts to a uniform signature
+/// </summary>
+interface IDemoScript
+{
+    /// <summary>
+    /// The one required static method for all demo scripts
+    /// </summary>
+    static abstract void RunDemo(bool showPlots = true);
```

Comment:
abstract is not supported yet in this version of .NET

Reply:
This is why I targeted .NET 7 (for this _consuming_ app only, not the published Vts library) - it's a way to add constraints on static methods, and generally improves things. If it's absolutely necessary to downgrade, that's fine. But any C# developer will have .NET 7 and I don't see a good reason to limit ourselves artificially when we're consuming the published/tested Vts.

Reply:
You can add preview to the language in csproj to keep that in place.

We stick with the LTS versions so we don't have to upgrade as often.

Reply:
Thanks for all the add'l comments. :) Ok, sounds good. Maybe if we can get automated CI/CD working, I can change your mind. 😇